README.md

```diff
@@ -30,4 +30,5 @@ There are optional arguments such as:
 
 - `--draft=true` To set the release as a draft. Default: `false`
 - `--prerelease=true` To set the release as a prerelease. Default: `false`
-- `--prefix=v` Add a prefix to the tag version `e.g. v1.0.1`
\ No newline at end of file
+- `--prefix=v` Add a prefix to the tag version `e.g. v1.0.1`
+- `--includeMessages=MergePull/Commits/All` used to filter the messages added to the release notes. Default: `Commits` 
```

Comment:
- Can we use `--include-messages` as camel case isn't that common in CLI programs.
- I'd rather the options to be: `merges/commits/all`